base/doc/usrguide.tex

```diff
@@ -858,6 +858,42 @@ \subsection{Typesetting verbatim-like material}
 Similarly, the \texttt{verbatim} environment sets up the meaning of \cs{par}
 suitable for breaking lines.
 
+\subsection{Verbatim environments}
+\label{sec:cmd:verbenv}
+
+In some cases, as well as grabbing an environment body you will want the
+contents to be treated verbatim. This is available using the argument
+specification~\texttt{f}. Like the \texttt{b} specification, this has to be the
+last one. Thus for example
+\begin{verbatim}
+\NewDocumentEnvironment{MyVerbatim}{!O{\ttfamily} +f}
+  {\begin{center} #1 #2\end{center}} {}
+\begin{MyVerbatim}[\ttfamily\itshape]
+  % Some code is show here
+  $y = mx + c$
+\end{MyVerbatim}
+\end{verbatim}
+will typeset
+\NewDocumentEnvironment{MyVerbatim}{!O{\ttfamily} +f}
+  {\begin{center} #1 #2\end{center}} {}
+\begin{MyVerbatim}[\ttfamily\itshape]
+  % Some code is show here
+  $y = mx + c$
+\end{MyVerbatim}
+
+As grabbing the entire contents verbatim, newlines are always permitted: if you
+use \texttt{f} rather than \texttt{+f}, a warning will be issued and \LaTeX{}
+will assume you meant to include the \texttt{+}. As for the \texttt{v}
+specification, new lines are stored as \cs{ObeyedLine}. In a similar fashion to
```

Comment:
In `ltnews41.tex` it is called `\obeyedline`, why it is called `\ObeyedLine` here?